src/ui/edit-content/EditContentForm.js

```diff
@@ -79,7 +91,23 @@ export class EditContentFormBody extends React.Component {
     onWillUnmount();
   }
 
+  setSection(sectionName, value) {
+    this.setState({
+      [sectionName]: value,
+    });
+  }
+
+  collapseSection(sectionName) {
+    const { [sectionName]: currentVisibility } = this.state;
+    this.setState({
+      [sectionName]: !currentVisibility,
```

Comment:
usually, when changing the state with a value that is already in the sate, we should use the function syntax to avoid async concurrences:

```
this.setState(state => ({ [sectionName]: !state.currentVisibility }))
```